samples/sensor/lsm303dlhc/src/main.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018, I-SENSE group of ICCS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <sensor.h>
+#include <stdio.h>
+#include <misc/util.h>
+
+s32_t read_sensor(struct device *sensor, enum sensor_channel channel)
+{
+	struct sensor_value val[3];
+	s32_t ret = 0;
+
+	ret = sensor_sample_fetch(sensor);
+	if (ret < 0) {
+		printf("Sensor sample update error\n");
+		goto end;
+	}
+
+	ret = sensor_channel_get(sensor, channel, val);
+	if (ret < 0) {
+		printf("Cannot read sensor channels\n");
+		goto end;
+	}
+
+	printf("( x y z ) = ( %f  %f  %f )\n", sensor_value_to_double(&val[0]),
+					       sensor_value_to_double(&val[1]),
+					       sensor_value_to_double(&val[2]));
+
+end:
+	return ret;
+}
+
+void main(void)
+{
+	struct device *accelerometer = device_get_binding("LSM303DLHC-ACCEL");
+	struct device *magnetometer = device_get_binding("LSM303DLHC-MAGN");
+
+	if (accelerometer == NULL) {
+		printf("Could not get LSM303DLHC-ACCEL device\n");
```

Comment:
If you agree with previous comment; printf("Could not get %s device\n", CONFIG_LSM303DLHC_ACCEL_NAME);

Reply:
Done